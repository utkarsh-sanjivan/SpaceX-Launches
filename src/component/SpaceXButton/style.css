.selected-color {
    background-color: #7dbb32;
    outline: none;
    width: 100%;
    height: 100%;
    border: none;
    border-radius: 5px;
    cursor: pointer;
    box-shadow: 4px 4px #999;
}

.unselected-color {
    background-color: #c5df9d;
    outline: none;
    width: 100%;
    height: 100%;
    border: none;
    border-radius: 5px;
    cursor: pointer;
}
