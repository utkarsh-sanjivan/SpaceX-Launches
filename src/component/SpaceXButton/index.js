import React from 'react';
import { Button } from 'antd';
import './style.css';

const SpaceXButton = props => (
    <Button 
        {...props}
        className={props.isSelected? 'selected-color': 'unselected-color'}
    />
)

export default SpaceXButton;