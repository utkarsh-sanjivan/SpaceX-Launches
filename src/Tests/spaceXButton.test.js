import React from 'react';
import { create } from "react-test-renderer";
import SpaceXButton from './../component/SpaceXButton/index';

test('Test SpaceX Button Component', () => {
  const component = create(<SpaceXButton 
        isSelected={true}
    />);
  expect(component.toJSON()).toMatchSnapshot()
});