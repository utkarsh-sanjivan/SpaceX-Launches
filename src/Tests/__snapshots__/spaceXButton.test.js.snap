// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test SpaceX Button Component 1`] = `
<button
  className="ant-btn selected-color"
  isSelected={true}
  onClick={[Function]}
  type="button"
/>
`;
